ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -47,15 +47,28 @@ function mirror_release_image_for_disconnected_upgrade() {
       ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
 MIRRORED_RELEASE_IMAGE=${DS_REGISTRY}/localimages/local-release-image
```

Comment:
Noticed a minor (pre-existing) thing: 
* `local-test-image` contains the mirrored images required by the tests
* `local-release-image` already contains the mirrored images by [dev-scripts](https://github.com/openshift-metal3/dev-scripts/blob/b8c619c1515d03f162adf08cbb89f9ba5c6d5cf1/common.sh#L196-L199) for the installation

Maybe here for slightly more clarity we could use a `local-upgrade-image` for mirroring the upgrade images.

Reply:
When I first introduced this mirroring, I exactly tried same thing. But it did not work. Now I realized that it did not work because of lack of imagecontentsourcepolicy and it should work now. I'll change that way, good point